### PR TITLE
Adding 13 new items and a DZE version of the vss vintorez

### PR DESCRIPTION
**Pull request by AirwavesMan**
I think the new items are a good addition for server admins. Most items were used by certain scripts, so a few new items are a good option to expand the possibilities. Admins should decide what items they want to use and can add that to the loot table then.
The new items are based of the Kostey items and are toolitems.

Ive added a new DZE version of the VSS Vintorez as well because in 1.0.5.1 the Vintorez spawns with a 20 rounds magazine.

### CODE REVIEW

File: SQF/dayz_code/stringtable.xml [hunk 1]
```diff
@@ -16858,6 +16858,119 @@
...
+		<Key ID="STR_EQUIP_NOTEBOOK_PHY_DESC">
+			<English>A notebook with many notes about physical connections.</English>
+			<German>Ein Notizbuch mit vielen Notizen über physikalische Zusammenhänge.</German>
+		</Key>	
+		<Key ID="STR_EQUIP_NOTEBOOK_SECRET">
+			<English>Secret Notebook</English>
+			<German>Geheimes Notizbuch</German>
+		</Key>
+		<Key ID="STR_EQUIP_NOTEBOOK_SECRET_DESC">
+			<English>In the notebook are some kind of secret information written. The text is encrypted with a secret code.</English>
```
Comment (ebayShopper) [inline SQF/dayz_code/stringtable.xml]: `are some kind of secret information`
should be
`is some kind of secret information`

File: SQF/dayz_code/stringtable.xml [hunk 2]
```diff
@@ -16858,6 +16858,119 @@
...
+			<German>In dem Notizbuch befinden sich offenbar geheime Informationen. Der Text ist mit einen Geheimcode verschlüsselt.</German>
+		</Key>	
+		<Key ID="STR_EQUIP_NOTEBOOK_HERBS">
+			<English>Herbs Notebook</English>
+			<German>Kräuter Notizbuch</German>
+		</Key>
+		<Key ID="STR_EQUIP_NOTEBOOK_HERBS_DESC">
+			<English>That notebook contains many information about herbs. It is very well documented and illustrated with pictures.</English>
```
Comment (ebayShopper) [inline SQF/dayz_code/stringtable.xml]: `That notebook contains many information about herbs.`
should be:
`The notebook contains a lot of information about herbs.`

File: SQF/dayz_code/stringtable.xml [hunk 3]
```diff
@@ -16858,6 +16858,119 @@
...
+		<Key ID="STR_EQUIP_DOCUMENT_RANDOM_DESC">
+			<English>Looks like a normal document, nothing special.</English>
+			<German>Sieht wie ein normales Dokument aus, nicht besonderes.</German>
+		</Key>	
+		<Key ID="STR_EQUIP_DOCUMENT_MILITARY">
+			<English>Military Document</English>
+			<German>Militärdokument</German>
+		</Key>
+		<Key ID="STR_EQUIP_DOCUMENT_MILITARY_DESC">
+			<English>A military document with many information about weapons, ammunition, tanks and helicopters.</English>
```
Comment (ebayShopper) [inline SQF/dayz_code/stringtable.xml]: `document with many information about`
should be
`document with a lot of information about`

File: SQF/dayz_code/stringtable.xml [hunk 4]
```diff
@@ -16858,6 +16858,119 @@
...
+		<Key ID="STR_EQUIP_MANUAL_REPAIR_DESC">
+			<English>A repair manual about how to fix items.</English>
+			<German>Ein Reperaturhandbuch über das Reparieren von Gegenständen.</German>
+		</Key>		
+		<Key ID="STR_EQUIP_MAP_Military">
+			<English>Military Map</English>
+			<German>Militärkarte</German>
+		</Key>
+		<Key ID="STR_EQUIP_MAP_Military_DESC">
+			<English>A military map with many markers. It looks like that the map was used for a big military operation.</English>
```
Comment (ebayShopper) [inline SQF/dayz_code/stringtable.xml]: `It looks like that the map was used for a big military operation.`
should be
`It looks like the map was used for a big military operation.`

**Comment by AirwavesMan** [comment]
Done

**Pull request merged**